test/gilded-rose.spec.ts

```diff
@@ -4,6 +4,115 @@ describe('Gilded Rose', () => {
   it('should foo', () => {
     const gildedRose = new GildedRose([new Item('foo', 0, 0)]);
     const items = gildedRose.updateQuality();
-    expect(items[0].name).toBe('fixme');
+    expect(items[0].name).toBe('foo');
   });
 });
+
+describe('Quality degrades twice as fast after sell by', () => {
+  it('should decrease 2x', () => {
```

Comment:
Don't think this is what describe blocks are meant for - they're meant for groups of `it` blocks that can be grouped in a meaningful way you can have all your `it` block blow in one single describe block.